web/src/containers/activity/Overview.js

```diff
@@ -0,0 +1,144 @@
+import { FormLabel, TextField } from '@cmsgov/design-system-core';
+import PropTypes from 'prop-types';
+import React, { useMemo, useCallback } from 'react';
+import { connect } from 'react-redux';
+
+import { t } from '../../i18n';
+import { updateActivity as updateActivityAction } from '../../actions/activities';
+import { RichText } from '../../components/Inputs';
+import Instruction from '../../components/Instruction';
+import { Subsection } from '../../components/Section';
+import { selectActivityByKey } from '../../reducers/activities.selectors';
+
+const Description = props => {
+  const { activity, updateActivity } = props;
+  const { alternatives, description, summary } = activity;
+
+  const overviewLabel = useMemo(
+    () =>
+      t('activities.overview.activityOverviewInput.label', {
+        defaultValue: 'Provide a short overview of the activity.'
+      }),
+    []
+  );
+  const overviewHint = useMemo(
+    () =>
+      t('activities.overview.activityOverviewInput.hint', {
+        defaultValue: ''
+      }),
+    []
+  );
+  const overviewOnChange = useCallback(
+    ({ target: { value } }) => updateActivity(activity.key, { summary: value }),
+    [activity.key]
+  );
+
+  const descriptionLabel = useMemo(
+    () =>
+      t(
+        `activities.overview.activityDescriptionInput.${
+          activity.fundingSource === 'HIE' ? 'hie' : 'standard'
+        }.label`,
+        {
+          defaultValue:
+            'Include as much detail as is necessary to explain the activity.'
```

Comment:
Just so I'm sure I follow: this is coming from the YAML in some cases, but in case it's not, we're defining the default value here?

Reply:
Correct, though I'm not sure it's necessary.  Maybe we'd prefer there be an error if these are missing from the YAML?  🤷‍♂